### ISSUE TITLE
Make the participant list table accessible

### ISSUE THREAD

**Issue by AlexKMarshall** [AlexKMarshall/regMan#51]
It should be an html table, not the divs it currently is to be usable with screenreaders.

If it stays as divs, it must be marked up with role attributes on every element
